waku/v2/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -282,9 +280,9 @@ proc sleep*(s: PostgresDriver, seconds: int):
   # database for the amount of seconds given as a parameter.
   try:
     let params = @[$seconds]
-    s.connection.exec(sql"SELECT pg_sleep(?)", params)
+    let _ = await s.connPool.query("SELECT pg_sleep(?)", params)
```

Comment:
```suggestion
    asyncCheck await s.connPool.query("SELECT pg_sleep(?)", params)
```

Reply:
https://forum.nim-lang.org/t/5279#33208

discard is OK for tests imo (in some situations)

Reply:
Thanks for the recommendation! However, the test got stuck with that.
Therefore, in the end I'll suggest sth like:
```code
    let sleepRes = await s.connPool.query("SELECT pg_sleep(?)", params)
    if sleepRes.isErr():
      return err("error in postgres_driver sleep: " & sleepRes.error)
```